docs/advanced/monitoring/applications-insights.md

```diff
@@ -54,8 +71,15 @@ namespace Example
             {
                 bus.Publish(new MyMessage{Value = "Hello, World."});
```

Comment:
Missing await ?

Reply:
not going to cry about it.

Reply:
Oh - it's definitely made me cry when i've missed them. I know it's not required here though :)